pkg/deploy/generator/resources.go

```diff
@@ -1513,6 +1513,27 @@ func (g *generator) rbac() []*arm.Resource {
 		},
 	}
 }
+func (g *generator) rbac_prod() []*arm.Resource {
```

Comment:
suggest: the function name should by golang-esque, `rbacProd` and avoid `_`.

Reply:
Corrected